iceoryx_hoofs/include/iceoryx_hoofs/cxx/helplets.hpp

```diff
@@ -217,153 +67,13 @@ static constexpr uint64_t arrayCapacity(T const (&/*notInterested*/)[CapacityVal
     return CapacityValue;
 }
```

Comment:
This is kind of a type trait

Reply:
No that's not a type trait. But I suppose it fits best into `primitives`.

Reply:
@elBoberido What do you think about moving it to `size.hpp` and renaming it to `iox::size`? From C++17 onwards it available in the STL anyway: https://en.cppreference.com/w/cpp/iterator/size

Reply:
Sounds good. Doesn't the `have` an implementation to check for all sizes independent of the string type? Maybe this could be consolidated but we have to take care of the null termination.

Reply:
@elBoberido Not sure what you mean with `have`.

Reply:
@mossmaurice I meant `Doesn't the 'string' have` :laughing: